docs/api/metric.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2022-Present Couchbase, Inc.
```

Comment:
3.0.x didn't have api specs, so we shouldn't need to worry about backporting these files/this commit.

Reply:
https://github.com/couchbase/sync_gateway/pull/6344/commits/8ad6cad10e38a859267a6d243766ec601d32615d